CHANGELOG.md

```diff
@@ -139,6 +139,7 @@
   API and added builders for customizing less common settings.][3516]
 - [Allow control of sort direction in `First` and `Last` aggregations.][3517]
 - [Implemented `Text.write`, replacing `File.write_text`.][3518]
+- [Removed `select`, `group`, `sort` and releated type from tables.][3519]
```

Comment:
```suggestion
- [Removed obsolete `select`, `group`, `sort` and releated types from tables.][3519]
```

I think it's worth noting that these functions were replaced by new stuff, not that we just remove them because so.